packages/css/src/plugins/sort-shorthand-declarations.ts

```diff
@@ -37,31 +37,19 @@ const sortNodes = (a: ChildNode, b: ChildNode): number => {
 
   if (!aDecl?.prop || !bDecl?.prop) return 0;
 
-  const aShorthand = shorthandFor[aDecl.prop as ShorthandProperties];
-  if (aShorthand === true || aShorthand?.includes(bDecl.prop)) {
-    return -1;
-  }
-
-  const bShorthand = shorthandFor[bDecl.prop as ShorthandProperties];
-  if (bShorthand === true || bShorthand?.includes(aDecl.prop)) {
-    return 1;
-  }
-
-  const aShorthandBucket = shorthandBuckets[aDecl.prop as ShorthandProperties];
-  const bShorthandBucket = shorthandBuckets[bDecl.prop as ShorthandProperties];
+  // Why default to Infinity? Because if the property is not a shorthand property,
+  // we want it to come after all of the other shorthand properties.
+  const aShorthandBucket = shorthandBuckets[aDecl.prop as ShorthandProperties] ?? Infinity;
```

Comment:
how the bucket system works is that every shorthand property is assigned a number

e.g. `padding` is 1, `padding-block` and `padding-inline` is 2

`padding-{left, top, right, bottom}` is assigned a number and so defaults to `Infinity`

when sorting, we ensure that properties with lower numbers come before properties with higher numbers

this also has the side effect of making the behaviour almost identical to runtime mode

Reply:
Yeah, this changed because mine originally was like "A is a shorthand of B", now it's more like "A is shorthand 5", "B is shorthand 3" and this change LGTM.

This doesn't sort `padding-block-start` vs. `padding-left`, but that's still fine, those are "siblings" and that's what we want.  I can't think of any scenario where this isn't what we want…maybe like `border-block-color` and `border-top` are sorted as siblings now and there could be some odd conflicts…?  But I'd say if we have issues with that we have like `sort-logical-properties` or something—that's separate to this and should rarely come up (but likely does exist).

Reply:
> This doesn't sort padding-block-start vs. padding-left, but that's still fine, those are "siblings" and that's what we want…maybe like border-block-color and border-top are sorted as siblings now and there could be some odd conflicts…?

I can add it to `packages/utils/src/shorthand.ts` if you want :) just let me know

Reply:
No, let's not, I don't know what that should even be. It's fine as-is, and if we need to add this later we have a good path forward!